### PR TITLE
Enhancement: Keep packages sorted in composer.json

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] keeps packages sorted in `composer.json`

💁‍♂️ For reference, see https://getcomposer.org/doc/06-config.md#sort-packages.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,31 +8,34 @@
             "email": "daniel@dantleech.com"
         }
     ],
+    "config": {
+        "sort-packages": true
```
Comment (localheinz) [inline composer.json]: After adding this configuration, I ran

```
$ composer require php:^7.0
```

and

```
$ composer require --dev phpunit/phpunit:^6.0
```

to trigger sorting.

**Comment by localheinz** [comment]
@dantleech 

Build is failing, to be honest, I have no idea how that could be related to the proposed changes!

**Comment by dantleech** [comment]
Build always fails for 3rd party PRs, it's realted to the PHAR build. Will merge and hope for the best :)

**Comment by dantleech** [comment]
Can always revert it

**Comment by localheinz** [comment]
Thank you, @dantleech!

**Pull request merged**